libs/extended/src/System/Logger/Extended.hs

```diff
@@ -1,12 +1,17 @@
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
 -- | Tinylog convenience things.
 module System.Logger.Extended
     ( mkLogger
     , mkLogger'
+    , LoggerT(..)
     ) where
 
 import Imports
 
 import qualified System.Logger as Log
+import qualified System.Logger.Class as LC
+import Database.CQL.IO
```

Comment:
I don't love that our logging lib now depends on cql; but I needed to define a MonadClient for `LoggerT`... The other option is to define LoggerT directly alongside the tests and define the instance there. Open to alternative ideas here if people have some 🤷‍♂️

Reply:
I would leave these considerations in the code as a comment, and then leave the code where it is so it's easier to find.